crates/services/gas_price_service/src/v1/uninitialized_task.rs

```diff
@@ -307,10 +374,12 @@ where
             block_gas_capacity,
             block_bytes,
             fee_wei.into(),
+            &mut tx,
         )?;
         let metadata: UpdaterMetadata = updater.clone().into();
```

Comment:
Not related to this PR, but I noticed that we clone the entire updater and then just pick up a couple of fields from the cloned one to construct metadata. I think we can consider cloning only the relevant fields, especially because we do it in a loop.